menu_silver.rb

```diff
@@ -0,0 +1,21 @@
+adjectives = ['hot', 'cold', 'creamy', 'soft', 'hard', 'crunchy', 'warm', 'spicy', 'gelatinous',
+  'chewy', 'crispy', 'juicy', 'tender', 'mushy', 'greasy', 'fluffy', 'frozen', ]
+
+styles = ['fried', 'steamed', 'baked', 'braised', 'roasted', 'boiled', 'pressure-cooked', 'broiled',
+  'freeze-dried', 'dehydrated', 'carmelized', 'stewed', 'pickled', 'seared',  'bar-b-qued', 'marinated',
+  'candied']
+
+foods = ['chicken','pork', 'shrimp', 'tilapia', 'beef', 'turkey', 'meatloaf', "pie", 'cookie', 'ice cream',
+  'donut', 'cake','mashed potatoes', 'hamburger', 'sausage', 'bananas', 'crab' ]
+
+prng = Random.new
+
+(0..foods.length-1).each do |i|
```

Comment:
Cool, you used an loop type has built in incrementation. :+1: 

Another way you could have done this is:
```
(foods.length-1).times do |i|
  ...
end
```